README.md

```diff
@@ -31,12 +31,20 @@ dependencies {
 }
 ```
 
+Then run:
+
+```shell
+gradle assemble
```

Comment:
Should probably use `./gradlew` or `gradlew.bat`. Otherwise, the user has to have gradle installed on their system.